src/components/ToolMenu/LayerTree/LayerTreeContextMenu/index.tsx

```diff
@@ -165,6 +207,18 @@ export const LayerTreeContextMenu: React.FC<LayerTreeContextMenuProps> = ({
     });
   }
 
+  if (downloadConfig) {
+    const downloadItems = downloadConfig.map((dlConfig: DownloadConfig) => {
+      return {
+        label: t('LayerTreeContextMenu.downloadLayer', {
+          formatName: dlConfig.formatName
```

Comment:
Is there a default for the `formatName` if not given?

Reply:
When it's a geoserver the default will be xml, so we could use that. If it is for some reason not a geoserver or another external link, the user can still overwrite it.

Reply:
see c3b9d8e8e1b217d50fa20dfe1c8fed84f459db36